backend/src/main/java/jonaszeihe/ninjin/model/User.java

```diff
@@ -13,6 +12,7 @@
 @Builder
 @Document(collection = "users")
 public class User {
-    @Id
+
```

Comment:
i would add the id field that mongo db create implicitly for you
```suggestion
@Id
private String id;
```